docs/conf.py

```diff
@@ -47,11 +50,11 @@ def _get_output(self):
 Command.get_output = _get_output
 
 
-def setup(app: Sphinx) -> None:
+def setup(app: Sphinx) -> None:  # noqa: ARG001
     here = Path(__file__).parent
     root, exe = here.parent, Path(sys.executable)
     towncrier = exe.with_name(f"towncrier{exe.suffix}")
     cmd = [str(towncrier), "build", "--draft", "--version", "NEXT"]
-    new = check_output(cmd, cwd=root, text=True)
+    new = check_output(cmd, cwd=root, text=True)  # noqa: S603
```

Comment:
We should probably ignore this rule globally. I have no idea what it's complaining about.

Reply:
I think locally is better to be explicit that this instance is allowed but not in general. https://beta.ruff.rs/docs/rules/#flake8-bandit-s shows the rules with explanation.

Reply:
Yes, I read the rule explanation. It seems to complain about every call to `subprocess`, as far as I can see. I tested running it with `cmd` set to a list of hardcoded strings, and it still raised an S603 error. That doesn't seem useful to me - have I missed something?